Library/ViewModels/PaymentMethodsViewModelTests.swift

```diff
@@ -10,18 +10,21 @@ import Prelude
 internal final class PaymentMethodsViewModelTests: TestCase {
 
   let vm = PaymentMethodsViewModel()
-  let paymentMethods = TestObserver<[GraphUserCreditCard.CreditCard], NoError>()
   let goToAddCardScreen = TestObserver<Void, NoError>()
+  let paymentMethods = TestObserver<[GraphUserCreditCard.CreditCard], NoError>()
+  let showAlert = TestObserver<String, NoError>()
+  let tableViewIsEditing = TestObserver<Bool, NoError>()
 
   internal override func setUp() {
     super.setUp()
 
-    self.vm.outputs.paymentMethods.observe(paymentMethods.observer)
     self.vm.outputs.goToAddCardScreen.observe(goToAddCardScreen.observer)
+    self.vm.outputs.paymentMethods.observe(paymentMethods.observer)
```

Comment:
a reference to `self` is missing here. (I think I was the one who forgot it 😅  )